app.py

```diff
@@ -25,11 +25,12 @@ def get_remember_thee():
 def sign_up():
 	try:
 		data = request.get_json()
-		subject = "User Sign Up: " + data['name']
+		subject = "User Sign Up: " + data['firstName', 'lastName']
```

Comment:
I don't believe this is proper syntax for [accessing](https://www.w3schools.com/python/python_dictionaries_access.asp) a python dictionary. Should probably be something like:
```
subject = "User Sign Up: " + data['firstName'] + " " + data['lastName']
```

Reply:
Totally! makes sense. thanks for reviewing